### DISCUSSION TITLE
Downloading from the Contributor Binaries website vs platform repo?

### DISCUSSION THREAD

**Discussion by crimsonduelist** [Eloston/ungoogled-chromium#2763] [General]
Sry if I missed this somewhere in the docs or issues but I see for example windows version binaries in the contributor binaries site
[](https://ungoogled-software.github.io/ungoogled-chromium-binaries/releases/windows/64bit/)
two releases behind the repo binaries.
I know from the docs 

> These binaries are provided by anyone who are willing to build and submit them

and from the windows repo docs

> Download binaries from the Contributor Binaries website

So are they the same and the website s section was simply not yet updated for the latest versions? i.e. the flow is basically someone d build and submit to the repo -> then that build also goes to the site?
Or Is there a difference between the contributor' s site and the repo versions?

**Comment by networkException** [comment]
Yes they are the same as from the releases page, but updating the contributors page is manual

**Comment by crimsonduelist** [comment]
Tnx for the confirmation, closing this!
